src/admin-views/settings/tickets-commerce/paypal-commerce/connect-with-paypal.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Connect with PayPal section for the Tickets Commerce > PayPal Commerce gateway settings.
+ *
+ * @since TBD
+ *
+ * @var bool   $account_is_connected [Global] Whether the account is connected.
+ * @var string $merchant_id          [Global] The merchant ID (if there is any).
+ * @var string $formatted_errors     [Global] The formatted account errors.
+ * @var string $guidance_html        [Global] The guidance HTML used when showing errors.
+ */
+
+// @todo Replace font awesome icon usages.
+
+?>
+
+<div id="give-paypal-commerce-account-manager-field-wrap">
```

Comment:
We need to remove references to Give in the code, IDs and classes.

Reply:
Leaving this for frontend implementation